tests/test-verify-pool-md.yml

```diff
@@ -0,0 +1,50 @@
+---
+
+  # Only when pool is raid
+- name: check RAID options
+  block:
+
+    - name: get information about RAID
+      command: "mdadm --detail /dev/md/{{ storage_test_pool.name }}-1"
+      register: storage_test_mdadm
+      changed_when: false
+      # Note: using "mdadm --detail {{ storage_test_pool.volumes[0]._device }}"
+      # causes mdadm segfault on RHEL even though it works on Fedora
```

Comment:
What is the value of `storage_test_pool.volumes[0]._device` when it segfaults? Is that a valid path to the device?

Reply:
Yes. The whole command looked like this:
`mdadm --detail /dev/mapper/vg1-lv1`
It worked for Fedora but segfaulted on all RHELs.

Reply:
Ah, because that is the wrong device. You want to pass it the PV device, not the LV device. You can see an example of listing the PVs in `tests/test-verify-pool-members.yml`.

Reply:
Segfault is not a very good input validation :-) If you can reproduce this behaviour, please report it.

Reply:
To be clear, this will work as-is for stacks created by the role, but it will not work for stacks created by the customer/user prior to running the role.

Reply:
@dwlehman: The fix is intended as a quick workaround. Since this call happens only within our test it should be safe.

Reply:
Oh, that's right. This is only in the tests.